### PR TITLE
[SUB-3270] workflow diagram and tables

### PR DESCRIPTION
**Pull request by jchima**
# What does this change do?
Added workflow diagram and tables

# Here's an awesome image for your troubles
image

# Please include a link to the Change Request or Agile Story here
https://submishmash.atlassian.net/browse/SUB-3270

### CODE REVIEW

File: UniversalCi.md [hunk 1]
```diff
@@ -46,7 +46,8 @@
 ```
 
 ### ✏️Pipeline workflow Diagram ###
-![image](https://joseph-project-files.s3.amazonaws.com/Universal+CI+Pipelines.png)
+![image](https://joseph-project-files.s3.amazonaws.com/Universal+Workflow.png)
```
Comment (tevans-submittable) [inline UniversalCi.md:49]: What are the trade-offs between placing these in S3 vs in the repo?

**Review by tevans-submittable — COMMENTED** [review]
Doc updates look good, but can we add a comment to the Run .Net test step as to why its commented out or bring it back?

**Comment by jchima** [comment]
> Doc updates look good, but can we add a comment to the Run .Net test step as to why its commented out or bring it back?

comment added, builds are failing when running .Net tests, so I commented it out to let the build continue till we get a working test case

**Review by chrisjdewitt — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**